src/parser/mod.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2017 Kisio Digital and/or its affiliates.
```

Comment:
minor: You could have named the file `src/parser.rs` instead of `src/parser/mod.rs`?